source/validate.cpp

```diff
@@ -80,6 +84,7 @@ spv_result_t spvValidateOperandValue(const spv_operand_type_t type,
                                      spv_diagnostic* pDiagnostic) {
   switch (type) {
     case SPV_OPERAND_TYPE_ID:
+    case SPV_OPERAND_TYPE_TYPE_ID:
```

Comment:
There's also: SPV_OPERAND_TYPE_MEMORY_SEMANTICS_ID, SPV_OPERAND_TYPE_SCOPE_ID.

Since you started working on this, I've reorganized the enumerants in spv_operand_type_t so these groupings are easier to see.